### ISSUE TITLE
HoloToolkit menu item not visible

### ISSUE THREAD

**Issue by CarpeTempus** [Microsoft/HoloToolkit-Unity#671]
Hello,
I followed the gettingStarted guide but after Step 2 there is no "HoloToolkit" menu item displayed in Unity. I tried to install it several times but unfortunately without success. I have tried to install HoloToolkit-Unity with Unity Version 5.5.0f3 (with Version 1.5.5.0 of HoloToolkit-Unity) and 5.6.0f3 (with Version 1.5.6. of HoloToolkit-Unity).
What am I doing wrong? Or is it a bug? Thank you for your help in advance.
Cheers,
CarpeTempus

**Comment by StephenHodgson** [comment]
Make sure you don't have any compile errors. Until these are resolved you won't see the new menu item after the package is imported.

**Comment by CarpeTempus** [comment]
Hmm, unfortunately the console did not came up with compile errors. Anyway, I noticed that a new version of Unity (5.6.1f1) and HoloToolkit-Unity (1.5.7)  was released and so I gave it a try and installed both. After creating a new empty project and import of the HoloToolkit-Unity the menu item is now displayed.
Next I will try to import step by step my project code into this new project..

Long story short: my problem is solved, thank you for your help :)
